client/src/app/components/followings-search/followings-search.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-followings-search',
+  templateUrl: './followings-search.component.html',
+  styleUrls: ['./followings-search.component.scss'],
+})
+export class FollowingsSearchComponent implements OnInit {
+  followings = [
+    { firstName: 'Marwan', lastName: 'Ayadi' },
+    { firstName: 'Razine', lastName: 'Bensari' },
+    { firstName: 'Arthur', lastName: 'Tourneyrie' },
+    { firstName: 'Alessandro', lastName: 'Kreslin' },
+    { firstName: 'David ', lastName: 'Pigeon' },
+    { firstName: 'Abdulrahim', lastName: 'Mansour' },
+    { firstName: 'Walter', lastName: 'Fleury' },
+    { firstName: 'Hossein', lastName: 'Noor' },
+    { firstName: 'Lindsay', lastName: 'Bangs' },
+    { firstName: 'Marwan', lastName: 'Ayadi' },
+    { firstName: 'Razine', lastName: 'Bensari' },
+    { firstName: 'Arthur', lastName: 'Tourneyrie' },
+    { firstName: 'Alessandro', lastName: 'Kreslin' },
+    { firstName: 'David ', lastName: 'Pigeon' },
+    { firstName: 'Abdulrahim', lastName: 'Mansour' },
+    { firstName: 'Walter', lastName: 'Fleury' },
+    { firstName: 'Hossein', lastName: 'Noor' },
+    { firstName: 'Lindsay', lastName: 'Bangs' },
+    { firstName: 'Marwan', lastName: 'Ayadi' },
+    { firstName: 'Razine', lastName: 'Bensari' },
+    { firstName: 'Arthur', lastName: 'Tourneyrie' },
+    { firstName: 'Alessandro', lastName: 'Kreslin' },
+    { firstName: 'David ', lastName: 'Pigeon' },
+    { firstName: 'Abdulrahim', lastName: 'Mansour' },
+    { firstName: 'Walter', lastName: 'Fleury' },
+    { firstName: 'Hossein', lastName: 'Noor' },
+    { firstName: 'Lindsay', lastName: 'Bangs' },
+  ]; // to be changed
+  followingSearch = '';
+
+  constructor(private router: Router) {}
+  ngOnInit(): void {}
+
+  navigateToFollowingProfile(following: string): void {
+    this.router.navigate(['/user-profile/' + following]);
+  }
+  isSearchedFollowings(following): boolean {
```

Comment:
I tried the search bar and realized that if you put first name + space + a part of the last name it will not show any results (for instance "hossein n" will return no results), if you could add logic for that case it will be more convenient for users of our application.

Reply:
for now the search is only a filter. When I will do the logic I may use the same search library that we use in stock search